### PR TITLE
Fix super-lint runs: Customise textlint, fix files and disable check of assets/scss

### PR DESCRIPTION
**Pull request by gwarf**
Customised textlint (removing replacements from node to Node.js)  and fix files.
Disable check of assets/scss

### CODE REVIEW

File: content/en/providers/operations-manuals/man01_how_to_publish_site_information/_index.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ of these MUST remain.
 
 | Attribute                   | Example                       | Schema                          | Notes                                                                                                                                                                         |
 | --------------------------- | ----------------------------- | ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| GlueSiteName                | RAL-LCG2                      | Free Text, No White Space       | Same as GOCDB name if in GOCDB, your choice.                                                                                                                                  |
+| GlueSiteName                | RAL-LCG2                      | Free Text, No whitespace       | Same as GOCDB name if in GOCDB, your choice.                                                                                                                                  |
```
Comment (enolfc) [inline content/en/providers/operations-manuals/man01_how_to_publish_site_information/_index.md:134]:
```suggestion
| GlueSiteName                | RAL-LCG2                      | Free text, no whitespace       | Same as GOCDB name if in GOCDB, your choice.                                                                                                                                  |
```

File: content/en/users/cloud-compute/ec3/apps/ecas/_index.md [hunk 2]
```diff
@@ -96,8 +96,8 @@ the general wizard steps include:
 
 ![Cluster summary](ecas-summary.png)
 
-When the front-end node of the cluster has been successfully deployed, you will
-be notified with the credentials to access via SSH.
+When the frontend Node.js of the cluster has been successfully deployed, you
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/ecas/_index.md:99]:
```suggestion
When the frontend node of the cluster has been successfully deployed, you
```

File: content/en/users/cloud-compute/ec3/apps/ecas/_index.md [hunk 3]
```diff
@@ -126,9 +126,9 @@ Last login: Mon Nov 18 11:37:29 2019 from torito.i3m.upv.es
 [root@oph-server ~]#
 ```
 
-Both the front-end and the working node are configured by Ansible. This process
-usually takes some time. You can monitor the status of the cluster configuration
-using the `is_cluster_ready` command-line tool:
+Both the frontend and the working Node.js are configured by Ansible. This
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/ecas/_index.md:129]:
```suggestion
Both the frontend and the working nodes are configured by Ansible. This
```

I don't like "working" nodes, feels wrong to me, but not the point of this PR

File: content/en/users/cloud-compute/ec3/apps/ecas/_index.md [hunk 4]
```diff
@@ -238,26 +238,27 @@ This section will show how to monitor the ECAS environment and the resource
...
-- The **infrastructure dashboard** provides information about the percentage
-  of CPU, RAM, SWAP and disk used on each node (the front-end and the working nodes).
+- The **infrastructure dashboard** provides information about the percentage of
+  CPU, RAM, SWAP and disk used on each Node.js (the frontend and the working
+  nodes).
 
-front-end node             |  working node
-:-------------------------:|:-------------------------:
-![Front-end infra metrics](ecas-infra-server.png)  |  ![wn infra metrics](ecas-infra-io.png)
+|                 frontend Node.js                  |            working Node.js             |
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/ecas/_index.md:254]:
```suggestion
|                 frontend node                     |            working node                |
```

**Review by enolfc — CHANGES_REQUESTED** [review]
Just some minor changes

**Comment by gwarf** [comment]
@enolfc: linting is now OK, there are some links error for nordugrid wiki that seems down, not sure if it's permanent or not, and for https://fgsg.ct.infn.it/egissod/web/ssod. Not sure about the status of this service. Maybe @glarocca knows about this.
Otherwise it's probably ready for final review and merge.

**Review by enolfc — APPROVED** [review]
LGTM

**Pull request merged**